bin/qlf_pipeline.py

```diff
@@ -212,42 +351,43 @@ def execute(self, camera, return_cameras):
             camera.get('duration')
         ))
 
-    def update_job(self, camera):
-        """ Update job and ingest QA results """
-
-        self.models.update_job(
-            job_id=camera.get('job_id'),
-            end=camera.get('end'),
-            status=camera.get('status')
-        )
-
         output_path = os.path.join(
             desi_spectro_redux,
-            self.data.get('output_dir'),
+            outputdir,
             'ql-*-%s-%s.yaml' % (
                 camera.get('name'),
-                self.data.get('zfill')
+                camera.get('zfill')
             )
         )
 
-        for product in glob.glob(output_path):
-            try:
-                qa = yaml.load(open(product, 'r'))
-
-                name = os.path.basename(product)
-                paname = qa['PANAME']
-                metrics = qa['METRICS']
-
-                logger.info("Ingesting %s" % name)
-                self.models.insert_qa(name, paname, metrics, camera.get('job_id'))
-            except Exception:
-                logger.error("Error ingesting %s" % name, exc_info=True)
-
-        logger.info("Finished ingestion of pipeline results for camera {}.".format(camera.get('name')))
-
-    def was_processed(self):
-        """ Returns [<Process object>] if expid was processed else returns [] """
+        self.models.update_job(
+            job_id=camera.get('job_id'),
+            end=camera.get('end'),
+            status=camera.get('status'),
+            output_path=output_path
+        )
 
-        expid = self.data.get('expid')
-        return self.models.get_expid_in_process(expid)
+if __name__ == "__main__":
+    exposure = {
+        'night': '20170428',
+        'expid': '3',
+        'zfill': '00000003',
+        'data_dir': '/home/singulani/raw_data',
```

Comment:
@crisingulani I would suggest removing hardwired paths even if this is for testing the python module. You could get all the required parameters as command line arguments, and add a comment showing an example on how to run the test.